orchagent/aclorch.cpp

```diff
@@ -345,11 +346,28 @@ bool AclRule::create()
     {
         SWSS_LOG_ERROR("Failed to create ACL rule");
         AclRange::remove(range_objects, range_object_list.count);
+        decreaseNextHopRefCount();
```

Comment:
In this case:
1. We're having two functions increaseNextHopRefCount() and decreaseNextHopRefCount() instead of one decreaseNextHopRefCount()
2. We're splitting our logic of taking of the reference and increasing the reference counters into two places.